cli/session.go

```diff
@@ -218,23 +218,31 @@ func DefaultArgsShellComp(cmd *cobra.Command, args []string, toComplete string)
 	return []string{}, cobra.ShellCompDirectiveNoFileComp
 }
 
-func ValidArgsGetTools(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+func ValidArgsGetTools(_ *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	if len(args) != 0 {
 		return nil, cobra.ShellCompDirectiveNoFileComp
 	}
 
 	return getTools(toComplete), cobra.ShellCompDirectiveNoFileComp
 }
 
-func ValidArgsGetMetrics(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+func ValidArgsGetMetrics(_ *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	if len(args) != 0 {
 		return nil, cobra.ShellCompDirectiveNoFileComp
 	}
 
 	return getMetrics(toComplete), cobra.ShellCompDirectiveNoFileComp
 }
 
-func getTools(toComplete string) []string {
+func ValidArgsGetCloudServices(_ *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	if len(args) != 0 {
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	}
+
+	return getCloudServices(toComplete), cobra.ShellCompDirectiveNoFileComp
+}
+
+func getTools(_ string) []string {
```

Comment:
```suggestion
func getTools() []string {
```

Reply:
This is intentional (future proof). The `toComplete` holds the typed input, we might want to include this in the request as a filter.